deploy/00_resolve_t_token.ts

```diff
@@ -6,19 +6,19 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { log } = deployments
   const { deployer } = await getNamedAccounts()
 
-  const KeepToken = await deployments.getOrNull("KeepToken")
+  const TToken = await deployments.getOrNull("TToken")
```

Comment:
Should it be `TToken` or `T`? I am looking at `threshold-network/solidity-contracts` [deployment script of the token staking contract](https://github.com/threshold-network/solidity-contracts/blob/4b363a6803389d0a83dc295e25218ee03ce8c925/deploy/07_deploy_token_staking.ts#L11) and T token is referenced by `const T = await deployments.get("T")` there. Also, manner T token artifacts are saved in [`T.json`](https://github.com/threshold-network/solidity-contracts/blob/main/deployments/mainnet/T.json).

Reply:
Done [6f2a9df](https://github.com/keep-network/coverage-pools/pull/203/commits/6f2a9dfef1239d00a8ff089c9069538cb92aaaed)